plugins/inputs/t128_peerpath/README.md

```diff
@@ -0,0 +1,128 @@
+# 128T Peer Path Input Plugin
```

Comment:
We need a better way to replicate this code. It's a tricky thing because the query is version dependent and we rely on the Monitoring Agent for all version dependent inputs. But it's not valid to configure an arbitrary GraphQL query in here as we will have post-processing logic specific to a query.

I think what we should have is this:
```toml
[[intpus.t128_peer_path]]
base_url = "..."
unix_socket = "..."
timeout = "..."

# versioned fields
exclude_hostname = false
```

And then for the implementation, it would look something like this:
```go
type T128PeerPath struct {
	CollectorName   string            `toml:"collector_name"`
	BaseURL         string            `toml:"base_url"`
	UnixSocket      string            `toml:"unix_socket"`
	Timeout         config.Duration   `toml:"timeout"`
	ExcludeHostname bool 			  `toml:"exclude_hostname"`
	gqlCollector *t128_graphql.T128GraphQL
}

// SampleConfig returns the default configuration of the Input
func (*T128PeerPath) SampleConfig() string {
	return sampleConfig
}

//Init sets up the input to be ready for action
func (plugin *T128PeerPath) Init() error {
	// parse the config
	plugin.gqlCollector.BaseURL = plugin.BaseURL
	plugin.gqlCollector.UnixSocket= plugin.UnixSocket
	// populate the gqlCollector fields etc.

	plugin.gqlCollector.Init()
}

// Gather takes in an accumulator and adds the metrics that the Input gathers
func (plugin *T128PeerPath) Gather(acc telegraf.Accumulator) error {
	metricsChan := make(chan telegraf.Metric)
	redirectAccumulator := agent.NewAccumulator(metricChan, /* other args*/)

	go func() {
		plugin.gqlCollector.Gather(redirectAccumulator)
		close(metricsChan)
	}()

	for  metric := range metricsChan {
		// conditional logic goes here to adjust the metric and decide if it should be included, then:
		acc.AddFields(metric /*args need to adjust*/)
	}
}

```